### DISCUSSION TITLE
Welcome to AsyncPywhatKit Discussions!

### DISCUSSION THREAD

**Discussion by SigireddyBalasai** [SigireddyBalasai/AsyncPywhatKit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by giulio333** [comment]
Hi! Can I make a instant message to single contact too?

**Reply by SigireddyBalasai** [reply]
Yes you can do it using send WhatsApp message instantly

**Reply by SigireddyBalasai** [reply]
Sorry for late

**Comment by giulio333** [comment]
I have problems 😔It doesn’t work yet.. can you write me the exact form to call the function?Inviato da iPhoneIl giorno 31 mar 2023, alle ore 07:45, SigireddyBalasai ***@***.***> ha scritto:﻿
Yes you can do it using send WhatsApp message instantly

**Reply by SigireddyBalasai** [reply]
sendwhatmsg_instantly is function it is asyncrous
